iset.mm

```diff
@@ -110246,6 +110385,89 @@ seq m ( + , ( n e. ZZ |-> if ( n e. A , [_ n / k ]_ B , 0 ) ) , CC )
         HWFZVUFYTVULWSFUYNUWPVUOVATHUWLUYNUQZUWMVUNUWOUYPWBUWLUYNKVRXAVUTEUWNUY
         OCUWLUYNUUEXDXBXERXFVKVUSYSUUAUUBAKLUUDUVGYMUUC $.
     $}
+
+    ${
+      $d A k m n x $.  $d B k n $.  $d F k m n x $.  $d H m x $.
+      $d K k m n x $.  $d M k m n x $.  $d N k n $.  $d f k n $.
+      $d k m n ph x $.
+      isummolem2a.dc $e |- ( ( ph /\ k e. ( ZZ>= ` M ) ) -> DECID k e. A ) $.
+      isummolem2a.g $e |- G = ( n e. NN |-> if ( n <_ ( # ` A ) ,
+        [_ ( f ` n ) / k ]_ B , 0 ) ) $.
+      isummolem2a.h $e |- H = ( n e. NN |-> if ( n <_ N ,
+        [_ ( K ` n ) / k ]_ B , 0 ) ) $.
+      summolem2.5 $e |- ( ph -> N e. NN ) $.
+      summolem2.6 $e |- ( ph -> M e. ZZ ) $.
+      summolem2.7 $e |- ( ph -> A C_ ( ZZ>= ` M ) ) $.
+      summolem2.8 $e |- ( ph -> f : ( 1 ... N ) -1-1-onto-> A ) $.
+      summolem2.9 $e |- ( ph -> K Isom < , < ( ( 1 ... ( # ` A ) ) , A ) ) $.
+      $( Lemma for summation.  (Contributed by Mario Carneiro, 3-Apr-2014.) $)
+      isummolem2a $p |- ( ph -> seq M ( + , F , CC )
+          ~~> ( seq 1 ( + , G , CC ) ` N ) ) $=
+        ( vm vx caddc cc cseq cfv c1 cli cuz cfz co wf1o wf clt wiso chash wceq
```

Comment:
This not the original ~summolem2a of set.mm, so "(Revised by Jim Kingdon, 3-Sep-2022.) $)" should be added, shoudn't it?

Reply:
Don't know if we want that every time there is some intuitionizing, but sure, this one seems major enough.

Updated.